### PR TITLE
Modpack definition helper

### PR DESCRIPTION
**Pull request by maros-klimovsky**
implementation and tests

### CODE REVIEW

File: ModpackDownloader.Tests/ModpackDefinitionHelperTests.cs [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+
+        [Test]
+        public void When_ZipFileIsMissingManifest_Then_ThrowException()
+        {
+            var helper = Kernel.Get<ModpackDefinitionHelper>();
+            Exception exception;
+
+             using(var archive = ZipFile.OpenRead(MISSING_MANIFEST_MODPACK_FILE))
+            {
```
Comment (adriana-smijakova) [inline ModpackDownloader.Tests/ModpackDefinitionHelperTests.cs]: aligning (for next time)

File: ModpackDownloader.Tests/ModpackDefinitionHelperTests.cs [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+                exception = Assert.Throws<Exception>(()=>helper.GetManifest(archive));
```
Comment (adriana-smijakova) [inline ModpackDownloader.Tests/ModpackDefinitionHelperTests.cs]: spaces around "=>"

File: ModpackDownloader/Manifest.cs [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    public class ModLoader
+    {
+        public string Id { get; set; }
+
+        
+        public bool Primary { get; set; }
+    }
+
+
+    public class FileType
```
Comment (adriana-smijakova) [inline ModpackDownloader/Manifest.cs]: Empty lines style is not in sync with other classes here.

File: ModpackDownloader/Manifest.cs [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    }
+
+
+    public class Minecraft
+    {
+        public string Version { get; set; }
+
+
+        public List<ModLoader> ModLoaders { get; set; }
+
```
Comment (adriana-smijakova) [inline ModpackDownloader/Manifest.cs]: Extra line.

File: ModpackDownloader/ModpackDefinitionHelper.cs [hunk 5]
```diff
@@ -0,0 +1,72 @@
+using System;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Net.Http;
+
+using Newtonsoft.Json;
+
+namespace ModpackDownloader {
```
Comment (adriana-smijakova) [inline ModpackDownloader/ModpackDefinitionHelper.cs]: Format curly brackets so its not "egyptian style".

**Review by adriana-smijakova — APPROVED** [review]
Just minor things.

**Comment by maros-klimovsky** [comment]
Closes #4.

**Pull request merged**